interfaces/apparmor/spec.go

```diff
@@ -31,6 +31,16 @@ import (
 	"github.com/snapcore/snapd/strutil"
 )
 
+// unconfined mode is only enabled when the interface supports it as a static
```

Comment:
UnconfinedMode describes the states of unconfined mode support, the mode ...

Reply:
Thanks, update in 0b7a92c01e